packages/create-audius-app/helpers/install.ts

```diff
@@ -0,0 +1,47 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import { yellow } from 'picocolors'
+import spawn from 'cross-spawn'
+
+/**
+ * Spawn an npm installation
+ *
+ * @returns A Promise that resolves once the installation is finished.
+ */
+export async function install(
+  /** Indicate whether there is an active Internet connection.*/
+  isOnline: boolean
+): Promise<void> {
+  const args: string[] = ['install']
+  if (!isOnline) {
+    console.log(
+      yellow('You appear to be offline.\nFalling back to the local cache.')
+    )
+    args.push('--offline')
+  }
+  /**
+   * Return a Promise that resolves once the installation is finished.
+   */
+  return new Promise((resolve, reject) => {
+    /**
+     * Spawn the installation process.
+     */
+    const child = spawn('npm', args, {
+      stdio: 'inherit',
+      env: {
+        ...process.env,
+        ADBLOCK: '1',
```

Comment:
interesting, sup with this?

Reply:
Apparently it disables all the requests for funding and job seeking in npm i output